hr_timesheet_sheet_autodraft/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2020 Brainbean Apps (https://brainbeanapps.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "HR Timesheet Sheet Auto-draft",
+    "version": "12.0.1.0.0",
+    "category": "Human Resources",
+    "website": "https://github.com/OCA/timesheet",
+    "author": "Brainbean Apps, " "Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "installable": True,
+    "application": False,
+    "summary": (
+        "Automatically draft a Timesheet Sheet for every time entry that does"
+        " not have a relevant Timesheet Sheet existing."
+    ),
+    "depends": ["hr_timesheet_sheet",],
```

Comment:
`,]` should be changed to `]` or linter will complain

Reply:
I have removed the trailing comma here and in all other places